src/main/java/dev/thedocruby/resounding/openal/Effect.java

```diff
@@ -0,0 +1,121 @@
+package dev.thedocruby.resounding.openal;
```

Comment:
Might want to consider putting this with the other effects in the `effect` package

Reply:
Potentially, though, this itself is not an effect.

Reply:
> this itself is not an effect.

Yes, but all of the classes in `effect` inherit from Effect, and in OOP langs that generally makes it good practice to include the base class in the package

Reply:
Alright, done.